src/js/lib/api/mock.js

```diff
@@ -1,13 +1,32 @@
 "use strict";
 
+//run this in terminal... USE_MOCK_API=true npm start
+//check index.js (this is why this command works)
+
+var userJSONObj={
+    	"user_name": "John Doe",
```

Comment:
Also ecma 6 has `let` and `const` so we don't need to use `var` here, you should be able to use `const` for this.